### PR TITLE
4_create

### PR DESCRIPTION
**Pull request by AnaScia**
I edited component CreateList. I added props.
I can reuse it in Packing and Budget now.

### CODE REVIEW

**Comment by AnaScia** [comment]
Can you check children please? I think it is wrong

File: src/component/CreateList.jsx [hunk 1]
```diff
@@ -29,11 +29,18 @@ function CreateList() {
...
         </button>
       </form>
       <div>
-        <Table save={save} setSave={setSave} />
+        <Table
+          save={save}
+          setSave={setSave}
+          column1={column1}
+          column2={column2}
+          column3={column3}
```
Comment (Alvacampos) [inline src/component/CreateList.jsx]: What if I have column4?

File: src/component/Table.jsx [hunk 2]
```diff
@@ -22,9 +22,9 @@ function Table({ save, setSave }) {
       <table>
         <thead>
           <tr>
-            <td>Item</td>
-            <td>Packed</td>
-            <td>Note</td>
+            <td>{column1}</td>
```
Comment (Alvacampos) [inline src/component/Table.jsx]: This should be dynamic, what if you have more than 3 columns? Try to pass down an array of items and use map to render all no matter how many they are.

File: src/view/Budget.jsx [hunk 3]
```diff
@@ -1,7 +1,16 @@
+import CreateList from '../component/CreateList';
+import '../styles/createList.css';
+
 function Budget() {
   return (
     <>
       <h1>Budget</h1>
+      <CreateList
+        boton="CREATE"
```
Comment (Alvacampos) [inline src/view/Budget.jsx]: boton or button? 🤣

File: src/view/Packing.jsx [hunk 4]
```diff
@@ -1,10 +1,14 @@
...
+import Table from '../component/Table';
 
 function Packing() {
   return (
     <>
-      <CreateList></CreateList>
-      {/*<Table></Table> */}
+      <h1>Packing</h1>
+      <CreateList boton="create" column1="Item" column2="Packed" column3="Note">
+        {/* le paso como children el input */}
```
Comment (Alvacampos) [inline src/view/Packing.jsx:10]: No need to comment this, you can delete it.

File: src/component/Table.jsx [hunk 5]
```diff
@@ -22,9 +22,10 @@ function Table({ save, setSave }) {
       <table>
         <thead>
           <tr>
-            <td>Item</td>
-            <td>Packed</td>
-            <td>Note</td>
+            {columns.map((column) => (
+              <td key={column}>{column}</td>
+            ))}
+            {/* <td>{columns}</td> */}
```
Comment (Alvacampos) [inline src/component/Table.jsx]: Delete this.

Comment (AnaScia) [reply src/component/Table.jsx]: Done

File: src/component/Table.jsx [hunk 6]
```diff
@@ -44,7 +44,8 @@ function Table({ save, setSave }) {
               </td>
               <td className="td_packed">
                 <div className="div_packed">
-                  <input type="checkbox" className="input-packed"></input>
+                  {/* <input type="checkbox" className="input-packed"></input> */}
```
Comment (Alvacampos) [inline src/component/Table.jsx]: This too 😄

Comment (AnaScia) [reply src/component/Table.jsx]: Okey

**Review by Alvacampos — APPROVED** [review]
LGTM 🚀

**Pull request merged**